src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient.Tests/IntTest/EFormidlingClientIntTest.cs

```diff
@@ -0,0 +1,266 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using Altinn.Common.EFormidlingClient.Configuration;
+using Altinn.Common.EFormidlingClient.Models;
+using Altinn.Common.EFormidlingClient.Models.SBD;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Logging.Debug;
+using Xunit;
+
+namespace Altinn.Common.EFormidlingClient.Tests.ClientTest
+{
+    /// <summary>
+    /// Represents a collection of int test, testing the<see cref="EFormidlingClientIntTest"/> class.
+    /// </summary>
+    public class EFormidlingClientIntTest : IClassFixture<EFormidlingClientIntTest.Fixture>
+    {
+        private readonly ServiceProvider _serviceProvider;
+        private string _guid;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EFormidlingClientIntTest"/> class.
+        /// </summary>
+        /// <param name="fixture">Fixture for setup</param>
+        public EFormidlingClientIntTest(Fixture fixture)
+        {
+            _serviceProvider = fixture.ServiceProvider;
+            _guid = fixture.CustomGuid;        
+        }
+
+        /// <summary>
+        /// Tests retrieving capabilities
+        /// </summary>
+        [Fact]
+        public async void Get_Capabilities() 
+        {        
+            var service = _serviceProvider.GetService<IEFormidlingClient>();
+            var result = await service.GetCapabilities("984661185");
+
+            Assert.Equal(typeof(Capabilities), result.GetType());
+            Assert.IsType<Capabilities>(result);
+        }
+
+        /// <summary>
+        /// Tests retrieving capabilities with invalid parameter input
+        /// </summary>
+        [Fact]
+        public async void Get_Capabilities_Invalid_ParameterInput()
+        {
+            var service = _serviceProvider.GetService<IEFormidlingClient>();
+            var result = await service.GetCapabilities("984661185");
+
+            ArgumentNullException ex = await Assert.ThrowsAsync<ArgumentNullException>(async () => await service.GetCapabilities(string.Empty));
+        }
+
+        /// <summary>
+        /// Tests sending Standard Business Document
+        /// </summary>
+        [Fact]
+        public async void Send_Standard_Business_Document()
```

Comment:
Please add a bit more of what we are actually testing or asserting here either in the summary or in the name of the function.